src/seedu/addressbook/commands/TagCommand.java

```diff
@@ -0,0 +1,57 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.common.Messages;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.person.UniquePersonList;
+import seedu.addressbook.data.tag.Tag;
+
+import java.util.HashSet;
+import java.util.Set;
```

Comment:
Imports from `java` should come before `seedu` :)

Reply:
Fixed